### ISSUE TITLE
Encryption functionality

### ISSUE THREAD

**Issue by VictorKabata** [a-sit-plus/signum#103]
First of all, awesome lib you have here.

Does the library provide encryption functionality after getting the Rsa public key from a X509 certificate?

Something like:
```kotlin
val cipher: Cipher = Cipher.getInstance("RSA/ECB/PKCS1Padding")
cipher.init(Cipher.ENCRYPT_MODE, rsaPublicKey)
val encryptedPayload: ByteArray = cipher.doFinal(initiatorPasswordByteArray)
```

**Comment by JesusMcCloud** [comment]
We're working on it (see draft pull request). but it is going to take a little, since we want to support attestation / deviceCheck and the possibility for remote crypto providers out of the box. You can still use the platform functionality in the meantime

**Comment by JesusMcCloud** [comment]
To be more precise: Once signing is there, asymmetric encryption comes easy-peasy. Symmetric encryption is planned for later. Since you are asking about asymmetric encryption, some time in September might work out for a release with that feature set. That depends on how fast we get signing going, though

**Comment by JesusMcCloud** [comment]
Signing is progressing nicely. ECDH key agreement and symmetric encryption is planned next. Asymmetric encryption might be possible in parallel. Out of curiosity: why do you want to RSA encrypt?

**Comment by JesusMcCloud** [comment]
RSA encryption has no ETA. Symmetric encryption will soon be released, though

**Comment by JesusMcCloud** [comment]
closing in favour  of #112
